README.md

```diff
@@ -1,2 +1,25 @@
-# open-inference-protocol
-Repository for storing open inference protocol schema
+# Open Inference Protocol Specification 
+
+The Open Inference Protocol(OIP) specification defines a standard protocol for performing machine learning model inference across
+serving runtimes for different ML frameworks. The protocol facilitates the implementation of a standardized and high performance data plane,
+promoting interoperability among model serving runtimes. The specification enables the creation of cohesive inference experience, 
+enpowering the development of versatile client or benchmarking tools that can work with all
+supported serving runtimes.
+
+- The inference REST [specification](./specification/protocol/inference_rest.md)
+- The inference gRPC [specification](./specification/protocol/inference_grpc.md)
+
+## Adoptions
+- KServe [v2 inference protocol](https://kserve.github.io/website/master/modelserving/data_plane/v2_protocol/)
+- NVIDIA [Triton inference server protocol](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/customization_guide/inference_protocols.html)
+- Seldon [MLServer v2 inference protocol](https://docs.seldon.io/projects/seldon-core/en/latest/reference/apis/v2-protocol.html)
```

Comment:
Could we also add Seldon Core V2 as a separate entry? We could point to this link: 

https://docs.seldon.io/projects/seldon-core/en/v2/contents/getting-started/#api-for-inference